apps/idos-enclave/src/lib/enclave.js

```diff
@@ -54,8 +54,8 @@ export class Enclave {
     };
   }
 
-  async keys() {
-    await this.ensurePassword();
+  async keys(currentUserPublicKey) {
```

Comment:
Careful! You missed the new argument when calling this on `async decrypt(fullMessage, senderPublicKey)`.